templates/tab/csharp/non-sso/Components/Welcome.razor

```diff
@@ -0,0 +1,64 @@
+﻿@using Microsoft.Extensions.Configuration
+@using Microsoft.AspNetCore.Hosting
+@using Microsoft.Extensions.Hosting
+@inject TeamsFx teamsfx
+@inject TeamsUserCredential teamsUserCredential
+@inject MicrosoftTeams MicrosoftTeams
+@inject IWebHostEnvironment HostEnvironment
+@inject IConfiguration Configuration
+@inject NavigationManager MyNavigationManager
```

Comment:
Are `teamsUserCredential` and `MyNavigationManager` used?

Reply:
Updated, thx!